root/io/io/CMakeLists.txt

```diff
@@ -5,4 +5,8 @@ RB_ADD_GBENCHMARK(IOBenchmarks
   )
  
  add_test(NAME rootbench-IOBenchmarks
-		COMMAND IOBenchmarks --benchmark_out_format=json)
\ No newline at end of file
+	COMMAND IOBenchmarks
+	--benchmark_out_format=json
```

Comment:
Since this boiler-plate is (likely) required for every single text, would it make sense to have a add_root_test that calls add_test which those extra parameters?

Reply:
Just doing it now, updating PR

Reply:
Done!